gravitee-apim-gateway/gravitee-apim-gateway-handlers/gravitee-apim-gateway-handlers-api/src/test/java/io/gravitee/gateway/reactive/handlers/api/processor/subscription/SubscriptionProcessorTest.java

```diff
@@ -289,6 +307,7 @@ void should_suffix_client_identifier_header_with_hash_when_subscription_equals_r
             verify(mockRequest).clientIdentifier(AdditionalMatchers.not(eq(TRANSACTION_ID)));
             verify(mockRequest).clientIdentifier(AdditionalMatchers.not(eq(REMOTE_ADDRESS)));
             assertThat(spyCtx.metrics().getClientIdentifier()).isEqualTo(clientIdentifier);
+            assertThat(cacheManager.getOrCreateCache(REMOTE_ADDRESS_HASHES_CACHE).get(REMOTE_ADDRESS)).isNotNull();
```

Comment:
You don't test TTL, I'm saying test the cache, I'm saying test that TLL works (maybe you need to tweak you class so you can change TTL values)

Reply:
I suppose you missed a word. Anyway ttl of the cache is tested on the cache module itself.